packages/braintree-utils/src/utils/is-braintree-accelerated-checkout-customer.spec.ts

```diff
@@ -1,4 +1,4 @@
-import isBraintreeAcceleratedCheckoutCustomer from './is-braintree-accelerated-checkout-customer';
+import isBraintreeFastlaneCustomer from './is-braintree-fastlane-customer';
 
 describe('isBraintreeAcceleratedCheckoutCustomer', () => {
```

Comment:
Please, rename test names and description

Reply:
renamed